compiler/rustc_target/src/spec/mod.rs

```diff
@@ -656,17 +662,24 @@ supported_targets! {
     ("armv7-unknown-linux-gnueabihf", armv7_unknown_linux_gnueabihf),
     ("thumbv7neon-unknown-linux-gnueabihf", thumbv7neon_unknown_linux_gnueabihf),
     ("thumbv7neon-unknown-linux-musleabihf", thumbv7neon_unknown_linux_musleabihf),
+    ("armv6-unknown-linux-dynmusleabihf", armv6_unknown_linux_dynmusleabihf),
```

Comment:
I believe Rust calls this `arm-...`, not `armv6-...`

Reply:
Seems so, yes.

Reply:
But I might be wrong, there appear to be other armv6 targets...

Reply:
On linux it appears to be `arm-`, so I changed the target to that to be aligned.